build/projects/Add-Ons (W1)/.AL-Go/customSettings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "ExternalAppDependencies" : [
+        "System Application",
+        "Business Foundation", 
+        "Base Application", 
+        "Application",
+        "Statistical Accounts",
+        "Sustainability"
```

Comment:
I assume this two apps are here because of they are needed for Demo Data?

Reply:
No, it's because the first two apps we're going to open-source take dependency on them :)